requirements/app.txt

```diff
@@ -11,3 +11,6 @@ rsa==4.9
 #better protobufs so i don't go insane.
 betterproto==1.2.5
 
+# local for macos (for now)
+file-read-backwards==3.0.0; sys_platform == 'darwin'
```

Comment:
we've been trying to keep the install size small because gog requires we pack each module we use along with our plugin, so if you can avoid using this, do so. That said, we're only at 3.5 Mb last i saw so this isn't blocking in any way. It's not worth seeking to the end of the file and manually reading lines backward when the module here does that.

Reply:
yeah this module is a pretty small one so it should be alr